src/components/pages/MapPage/index.js

```diff
@@ -1,39 +1,85 @@
-import React, { useState, useEffect } from "react";
+import React, { useState, useEffect, useRef } from "react";
 
 import GoogleMapReact from "google-map-react";
 import styled from "styled-components";
 
 import { getFeedInfo } from "../../../api";
 import theme from "../../../theme/theme";
-import { Icon } from "../../atoms";
+import { Icon, GpsIcon } from "../../atoms";
 import { MapTemplate } from "../../templates";
 
+import useSocket from "../../../hooks/useSocket";
+
 const StyledIcon = styled(Icon)`
   color: ${(props) => props.color};
 `;
 
 function MapPage() {
+  function getRandom() {
+    const rand09 = Math.floor(Math.random() * 10);
+    return rand09;
+  }
+
+  const random = `0.000${getRandom()}`;
+  const [socket, disconnect] = useSocket("map");
   const [defaultProps, setDefaultProps] = useState(null);
   const [zoomLevel, setZoomLevel] = useState(14);
   const [boundary, setBoundary] = useState({});
   const [feedLocation, setFeedLocation] = useState([]);
+  // const userGps = useRef();
+  const [userGps, setUserGps] = useState();
+  const [gpsLocations, setgpsLocations] = useState({});
 
   function spreadFeeds() {
     return feedLocation.map((feed) => {
       const { _id, image, cleaned, coordinates } = feed;
       const [longitude, latitude] = coordinates;
       const iconType = cleaned ? "leaf" : "trashCanFill";
       const color = cleaned ? theme.colors.green_1 : theme.colors.red;
+      return <StyledIcon key={_id} lat={latitude} lng={longitude} icon={iconType} color={color} size="sm" />;
```

Comment:
```suggestion
      const color = cleaned ? theme.colors.green_1 : theme.colors.red;
      
      return <StyledIcon key={_id} lat={latitude} lng={longitude} icon={iconType} color={color} size="sm" />;
```
개행을 해주시면 가독성에 조금 더 좋을 것 같습니다.😀